NguyenVanViet2004/FE-GlowUp
# Improve time picker compoennt for better user experience

## Related issues
Resolved https://github.com/NguyenVanViet2004/FE-GlowUp/issues/88

## Summary
- Updated the TimePicker component to improve the user interface and usability.
   - Implemented staggered alignment for even, odd, and half slots to enhance readability.
   - Added getItemLayout to fix scrollToIndex errors and improve scrolling performance.
   - Updated the FlatList behavior to ensure items appear in distinct positions when first rendered.
- Enhanced the overall user experience with a clean, consistent design.
## How to test
1. Navigate to the TimePicker component in the app.
2. Check that the time slots (even, odd, half) are displayed in a staggered layout when the component renders.
3. Scroll through the lists and confirm smooth scrolling behavior.
4. Verify no errors are thrown when selecting or scrolling to specific items.
5. Test across different devices and orientations to ensure layout consistency.

## Concern points

## Screenshots
![image](https://github.com/user-attachments/assets/c18d1fd2-6081-4c40-b194-81f0415a6972)
